commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java

```diff
@@ -96,7 +96,9 @@
 	CPC_QUALITY_MEASURE_ID_STRATA_MISMATCH(49,"Amount of stratifications `(Current number of "
 			+ "Reporting Stratifiers)` does not meet expectations "
 			+ "`(Number of stratifiers required)` for `(Current subpopulation type)` measure "
-			+ "`(Current Subpopulation UUID)`. Expected strata: `(Expected strata uuid list)`", true),
+			+ "`(Current Subpopulation UUID)`. Expected strata: `(Expected strata uuid list)`."
```

Comment:
Add a space before `Please refer to the`.

Reply:
This change has been made.

Reply:
Not sure if you missed a commit?  Github doesn't show a new commmit.

Reply:
Oh. I noticed I didn't push the changes. Sad day....